### ISSUE TITLE
App metrics error - model_count does not show any values

### ISSUE THREAD

**Issue by david-kn** [nautobot/nautobot-plugin-capacity-metrics#17]
### Environment
* Python version:  any
* Nautobot version:  1.0.1
* nautobot-capacity-metrics version:  1.0.1

### Expected Behavior
I expected to see nautobot_model_count for IPAM or DCIM modules

### Observed Behavior
There are no metrics shown for **nautobot_model_count** section.

### Steps to Reproduce
1. install this plugin according to the docs / modify configuration file

- extended list of PLUGINS  
`PLUGINS = ["nautobot_capacity_metrics"]`

- added configuration 

```
PLUGINS_CONFIG = {
   "nautobot_capacity_metrics": {
     "app_metrics": {
       "models": {
          "nautobot.dcim": {"Site": True, "Rack": True, "Device": True,},
          "nautobot.ipam": {"IPAddress": True, "Prefix": True},
        },
        "jobs": True,
        "queues": True,
       }
     }
   }
```

-  Everythins seems to work - the app and rq metrics are shown **except** the model_count 

![image](https://user-images.githubusercontent.com/14314517/125797445-0fd91e80-c190-42a7-bf65-a4ded40a1b48.png)

**Comment by dgarros** [comment]
@david-kn Please can you try this format for the config, we might need to update the documentation
```
    "nautobot_capacity_metrics": {
        "app_metrics": {
            "models": {
                "dcim": {
                    "Site": True,
                    "Rack": True,
                    "Device": True,
                    "Interface": True,
                    "Cable": True,
                },
                "ipam": {"IPAddress": True, "Prefix": True},
            },
            "queues": True,
        }
    },
```

**Comment by david-kn** [comment]
@dgarros Cool, this format seems to work.. Thanks.
...and I just added again also the _jobs_ part as well - I guess it was just left out :-) 


  ```
  "nautobot_capacity_metrics": {
        "app_metrics": {
            "models": {
                "dcim": {
                    "Site": True,
                    "Rack": True,
                    "Device": True,
                    "Interface": True,
                    "Cable": True,
                },
                "ipam": {"IPAddress": True, "Prefix": True},
            },
            "queues": True,
            "jobs": True
        }
    },

```

**Comment by nniehoff** [comment]
@david-kn is this fixed by #18?

**Comment by FragmentedPacket** [comment]
@nniehoff It doesn't appear so. I'll be submitting an PR shortly to fix this. The default doesn't appear to work either. Do we need to create an issue to track these separately?
